tools/clang/lib/SPIRV/LowerTypeVisitor.cpp

```diff
@@ -630,6 +630,10 @@ const SpirvType *LowerTypeVisitor::lowerResourceType(QualType type,
 
   assert(type->isStructureOrClassType());
 
+  if (isRasterizerOrderedView(type)) {
+    emitError("rasterizer ordered views are unimplemented", srcLoc);
```

Comment:
Running this command:
```
./build/bin/dxc -T ps_6_6 -E main -fcgl -spirv tools/clang/test/CodeGenSPIRV_Lit/type.rasterizer-ordered-buffer.unimplemented.hlsl
```

causes DXC to show the error (expected), but also the validator to complain the generated SPIR-V is invalid and to report this bug. Is that expected?